studio/src/mas-side-nav.js

```diff
@@ -77,19 +81,17 @@ class MasSideNav extends LitElement {
                     label="Home"
                     value="home"
                     @click="${navigateToPage('welcome')}"
-                    selected
+                    ?selected=${this.currentPage.value === 'welcome'}
```

Comment:
Please use `Store.page`. We discussed pros & cons of using StoreController for accessing data and the outcome was to use `Store.` api directly. 
Thus, we keep also automatic code completion.

Reply:
Addressed